cases/arithmetic_decimal/bitwise_not_decimal.yaml

```diff
@@ -0,0 +1,75 @@
+base_uri: https://github.com/substrait-io/substrait/blob/main/extensions/substrait/extensions/functions_arithmetic_decimal.yaml
+function: bitwise_not
+cases:
+  - group:
+      id: basic
+      description: Basic examples without any special cases
+    args:
+      - value: 0
+        type: decimal<38, 0>
```

Comment:
should we do `decimal<1, 0>`?  Make it more precise, like we did to all the other decimal yamls.